packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -46,6 +46,19 @@ export const build = (projectStructure: ProjectStructure): Plugin => {
           },
         },
         define: processEnvVariables(projectStructure.envVarPrefix),
+        experimental: {
+          renderBuiltUrl(filename, { hostType }) {
+            // Assets are returned with a leading slash for some reason. This adjusts the
```

Comment:
https://vitejs.dev/config/shared-options.html#base

They're returned with a leading slash because the default base is `/`